exonum-java-binding-fakes/src/test/java/com/exonum/binding/fakes/services/transactions/SetEntryTransactionIntegrationTest.java

```diff
@@ -44,12 +47,18 @@ static void loadLibrary() {
   @Test
   void executePutsTheValueIntoEntry() throws CloseFailuresException {
```

Comment:
Shan't this test also verify newly added behaviours?

Reply:
added

Reply:
There are no tests of the new behaviour :upside_down_face: Please use the assertion below as a template: read the expected entries from the database and verify that they match expected (which _must_ be explicitly set up in the context, because they matter in this test).